challenges/objects-arrays.js

```diff
@@ -85,36 +133,46 @@ zooAnimals = [{"animal_name":"Jackal, asiatic","population":5,"scientific_name":
 
 /* Request 1: .forEach()
 
-The zoo wants to display both the scientific name and the animal name in front of the habitats.  Return an array with only the animal and scientific names in it.  The individual values in the array should look like this "Name: Jackal, asiatic, Scientific: Canis aureus."
+ The zoo wants to display both the scientific name and the animal name in front of the habitats.  Return an array with only the animal and scientific names in it. 
+ The individual values in the array should look like this "Name: Jackal, asiatic, Scientific: Canis aureus."
 
 */
-const animalNames = [];
+const animalNames = []; 
+
+zooAnimals.forEach(animal => animalNames.push(`Name: ${animal.animal_name}, Scientific: ${animal.scientific_name}`))
+
 console.log(animalNames);
 
 /* Request 2: .map()    
 
-The zoos need a list of all their animal's names (names only, not scientific) converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
+ The zoos need a list of all their animal's names (names only, not scientific) converted to lower case. 
+ Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
 
 */
 
-const lowerCase = [];
+const lowerCase = [];  
+zooAnimals.map(animal => lowerCase.push(animal.animal_name.toLowerCase()));
```

Comment:
You should consider assigning zooAnimals.map to lowerCase instead of pushing into lowerCase. map creates a new array, so you don't need an empty array to push into.